styles/components/_modalWindow.scss

```diff
@@ -0,0 +1,159 @@
+// Positioning /
+// Display & Box Model /
+// Color /
+// Text /
+// Other /
+
+.modal {
+    // Positioning /
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    z-index: 1;
+    // Display & Box Model /
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    // Color /
+    background-color: rgba(0,0,0,.8);
+    &__container {
+        // Display & Box Model /
+        height: 889px;
+        width: 1200px;
+        border-radius: 16px;
+        padding: 3rem 4rem;
+        display: flex;
+        flex-direction: column;
+        // Color /
+        background: #FFFFFF;
```

Comment:
do you have this color set as scss variable? 🤔

Reply:
Yes, of course we do!